HOWTO.md

```diff
@@ -0,0 +1,115 @@
+# How To use 2DECOMP&FFT
+Full informations on how to use the 2DECOMP&FFT library together with some related theory are available [here](https://2decomp-fft.github.io/).
+
+The 2D Pencil Decomposition API is defined in three Fortran modules which should be used by applications as:
+```
+  use decomp_2d_constants
+  use decomp_2d_mpi
+  use decomp_2d
+```
+where ``use decomp_2d_constants`` defines all the parameters, ``use decomp_2d_mpi`` introduces all the MPI 
+related interfaces and ``use decomp_2d`` cointains the main decomposition and transposition APIs. The library is initialized using
+```
+  call decomp_2d_init(nx, ny, nz, p_row, p_col)
+```
+where ``nx``, ``ny`` and ``nz`` are the size of 3D global data to be distributed over
+a 2D processor grid :math:`p_row \times p_col`.
+Note that none of the dimensions need to be divisible by ``p_row`` or ``p_col``, i.e. the library can handle non-evenly distributed data.
```

Comment:
Remove that sentence, it is not always the case depending on the MPi library

Reply:
Ok done